sim/dex.ts

```diff
@@ -826,6 +826,22 @@ export class ModdedDex {
 				ruleTable.set(ruleSpec, '');
 				continue;
 			}
+			if (ruleSpec.split(":").length > 1) {
```

Comment:
This is an unnecessarily inefficient way to write `ruleSpec.includes(":")`

Reply:
Didn't know .includes existed, will change along with fixing build errors